modules/svg/osmose.js

```diff
@@ -0,0 +1,265 @@
+            drawLayer.selectAll('.qa_error.osmose')
+                .remove();
+            touchLayer.selectAll('.qa_error.osmose')
+                .remove();
+        }
+    }
+
+
+    // Enable the layer.  This shows the errors and transitions them to visible.
+    function layerOn() {
+        // Strings supplied by Osmose fetched before showing layer for first time
+        // NOTE: Currently no way to change locale in iD at runtime, would need to re-call this method if that's ever implemented
+        // FIXME: If layer is toggled quickly multiple requests are sent
+        // FIXME: No error handling in place
+        getService().loadStrings(editOn);
+
+        drawLayer
+            .style('opacity', 0)
+            .transition()
+            .duration(250)
+            .style('opacity', 1)
+            .on('end interrupt', function () {
+                dispatch.call('change');
+            });
+    }
+
+
+    // Disable the layer.  This transitions the layer invisible and then hides the errors.
+    function layerOff() {
+        throttledRedraw.cancel();
+        drawLayer.interrupt();
+        touchLayer.selectAll('.qa_error.osmose')
+            .remove();
+
+        drawLayer
+            .transition()
+            .duration(250)
+            .style('opacity', 0)
+            .on('end interrupt', function () {
+                editOff();
+                dispatch.call('change');
+            });
+    }
+
+
+    // Update the error markers
+    function updateMarkers() {
+        if (!_osmoseVisible || !_osmoseEnabled) return;
+
+        var service = getService();
+        var selectedID = context.selectedErrorID();
+        var data = (service ? service.getErrors(projection) : []);
+        var getTransform = svgPointTransform(projection);
+
+        // Draw markers..
+        var markers = drawLayer.selectAll('.qa_error.osmose')
+            .data(data, function(d) { return d.id; });
+
+        // exit
+        markers.exit()
+            .remove();
+
+        // enter
+        var markersEnter = markers.enter()
+            .append('g')
+            .attr('class', function(d) {
+                return [
+                    'qa_error',
+                    d.service,
+                    'error_id-' + d.id,
+                    'error_type-' + d.error_type,
+                    'item-' + d.item
+                ].join(' ');
+            });
+
+        markersEnter
+            .append('polygon')
+            .call(markerPath, 'shadow');
+
+        markersEnter
+            .append('ellipse')
+            .attr('cx', 0)
+            .attr('cy', 0)
+            .attr('rx', 4.5)
+            .attr('ry', 2)
+            .attr('class', 'stroke');
+
+        markersEnter
+            .append('polygon')
+            .attr('fill', d => getService().getColor(d.item))
+            .call(markerPath, 'qa_error-fill');
+
+        markersEnter
+            .append('use')
+            .attr('transform', 'translate(-5.5, -21)')
```

Comment:
The icons aren't quite centered in the pin head. This is probably a pre-existing problem with OSM and ImproveOSM markers too.

![Screen Shot 2020-02-04 at 2 02 24 PM](https://user-images.githubusercontent.com/2046746/73777587-bd302f80-4757-11ea-944f-572d5fe4ddc6.png)